go.mod

```diff
@@ -8,8 +8,10 @@ require (
 	github.com/cosmos/cosmos-sdk v0.34.4-0.20200530180557-ba70f4d4dc2e
 	github.com/gorilla/mux v1.7.4
 	github.com/pkg/errors v0.9.1
+	github.com/shopspring/decimal v1.2.0
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/viper v1.7.0
 	github.com/tendermint/tendermint v0.33.4
 	github.com/tendermint/tm-db v0.5.1
+	gopkg.in/validator.v2 v2.0.0-20200605151824-2b28d334fa05
```

Comment:
The project already uses a validator https://github.com/asaskevich/govalidator, please remove this and use the old one

Reply:
Done. now using asaskevich/govalidator